proof-systems/src/darlin/marlin/src/lib.rs

```diff
@@ -123,16 +126,31 @@ impl<'a, G: Group, PC: PolynomialCommitment<G>> Marlin<G, PC> {
             .map(|c| c.commitment().clone())
             .collect();
 
-        let vk_hash = D::digest(&serialize_no_metadata![index.index_info, index_comms]
-            .map_err(|e| {
-                Error::Other(format!("Unable to serialize vk elements: {:?}", e))
-            })?)
-            .as_ref()
-            .to_vec();
+        let vk_hash = D::digest(
```

Comment:
I guess lagrange_comms should be part of the vk digest too no ?

Reply:
As discussed offline with @UlrichHaboeck75, should not be necessary.